crates/wasi-common/src/path.rs

```diff
@@ -51,6 +47,9 @@ pub(crate) fn get(
     // any symlinks we encounter are processed by pushing them on the stack.
     let mut path_stack = vec![path.to_owned()];
 
+    // No longer need a borrow into the guest memory for `path`:
+    drop(path);
```

Comment:
Is that necessary? Could we scope it instead to do auto-dropping?

Reply:
this doesn't turn out to be necessary at all, idk what I was thinking. thanks for catching it.